docs/Accessibility.md

```diff
@@ -2,10 +2,23 @@
 
 ## Preamble
 
-GTFS and Netex define accessibility primarily in terms of binary access for wheelchair users: it's
-either on or off. Whilst it is the desire of the OTP developers to broaden the scope of
-accessibility the lack of data limits us to use this definition in the implementation and in this
-document.
+In this document and in OTP, the term "accessibility" is used with what has become its most common 
+meaning: design of products, devices, services, vehicles, or environments to ensure they are usable by 
+people with disabilities. It has other meanings in other contexts (see below).
+While accessibility is a complex subject, at this point GTFS and Netex mostly represent it very
+simply as a yes/no possibility of wheelchair use. While OTP developers hope to broaden the
+scope and nuance of accessibility support in OTP, the lack of detailed data from data producers
+currently limits implementation and discussion in this document to this binary 
+"wheelchair accessible" definition.
+
+The term "accessibility" has a completely separate, unrelated definition in the fields of
```

Comment:
The audience is people who search for "OTP accessibility" when looking for analysis uses and land on this page of the docs. People who are trying to understand the situation with travel time surfaces and isochrones, and ending up on the wrong page. It is indeed a bit out of place here.

The ideal would be to expand and revise the other page that is specifically about analysis uses. That other page will probably just redirect people to other tools (after giving them some background in case they're following up on past projects / articles).  Then this page (on the more common meaning of accessibility) would only need a single sentence along the lines of "if you're looking for the other meaning of accessibility, see analysis page (link)".

So I'll just move this text over to the analysis page and suggest a short link here.

Reply:
I think that would be great.

Reply:
This update to the analysis docs page has long been on my mind since we talked about it in the past. But my sense in meetings is always that the fate/state of isochrone and one-to-many functionality is still being decided and I am not yet sure of those final decisions. But I think we can get this finalized soon.

Reply:
Entur have publicly stated that they want to remove it, but haven't actually done it.